Composer/packages/extension/src/extensionContext.ts

```diff
@@ -103,6 +103,7 @@ class ExtensionContext {
     }
   }
 
+  /** @deprecated */
   public async loadPluginsFromFolder(dir: string) {
```

Comment:
Is there anything using this that would benefit from knowing that this is deprecated? If it's only used internally can we just remove it?

Reply:
It's used in a single test file. I'll clean it up another time.